source/includes/usage-examples/code-snippets/FindOneLegacy.java

```diff
@@ -0,0 +1,43 @@
+package usage_examples;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import com.mongodb.MongoClientURI;
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.Projections;
+import com.mongodb.client.model.Sorts;
+import com.mongodb.client.MongoCollection;
+
+public class FindOne
+{
+    public static void main( String[] args )
+    {
+        // Replace the uri string with your MongoDB deployment's connection string
+        MongoClientURI uri = new MongoClientURI("mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority");
+
+        MongoClient mongoClient = new MongoClient(uri);
+
+        MongoDatabase database = mongoClient.getDatabase("sample_mflix");
```

Comment:
Having a legacy example for every example is going to result in a lot of almost identical examples.  Notice though that from line 24 on, the examples are identical.  It's just the first two lines that are different.

Have you considered other approaches?  Perhaps we just have a legacy set of pages for configuring your MongoClient, and then have all the CRUD examples use the new API.  Or at least figure out to share most of the code and then dynamically create both regular and legacy examples.

Reply:
My preference is not to provide examples using the legacy API, but from your comment [here](https://github.com/mongodb/docs-java/pull/3#discussion_r405642314), I thought you wanted examples for both in this section.

I'd be happy to pursue another direction such as creating a page or section of a page that describes the differences between the two and then linking to that on each usage example page. Other than the construction of the client, are there any other differences between the legacy and current API?

Reply:
I wasn't clear enough in that comment.  Sorry about that.

There is no difference once you call `getDatabase()` on the client (the MongoClient types _themselves_ are different: the legacy is `com.mongodb.MongoClient`, the new is `com.mongodb.client.MongoClient` ), that's why I'm suggesting this approach.

Reply:
Sounds good -- I'll create a new ticket for creating the Legacy API differences, link to those differences on each usage example page, and remove the legacy API example files from this PR (and any other Usage Example PR).

Reply:
new ticket: https://jira.mongodb.org/browse/DOCSP-9930